test/tabs/test-firefox-tabs.js

```diff
@@ -1234,6 +1234,33 @@ exports['test active tab properties defined on popup closed'] = function (assert
   });
 };
 
+// related to bug 922956
+// https://bugzilla.mozilla.org/show_bug.cgi?id=922956
+exports["test ready event after window.open"] = function (assert, done) {
+  setPref(OPEN_IN_NEW_WINDOW_PREF, 2);
+  setPref(DISABLE_POPUP_PREF, false);
+
+  let firstRun = true;
+  tabs.on('ready', function onReady(tab) {
+    if (firstRun) {
+      assert.pass("tab ready callback after 1st window.open");
+      firstRun = false;
+      tab.close();
+    }
+    else {
+      assert.pass("tab ready callback after 2nd window.open");
```

Comment:
wouldn't the test pass even if neither of the `assert.pass` lines were ever called?

how does this line get called? won't the test exit on the first call to `assert.pass`? (i'm not sure exactly what testing framework is being used here, but `assert.pass` will end a test as passing in most frameworks i've seen)